tests/scripts/run_tests.py

```diff
@@ -2,7 +2,7 @@
 from sys import path
 
 tests_path = dirname(realpath(__file__))
-scripts_path = realpath(pjoin(tests_path, "..", "..", "scripts"))
+scripts_path = realpath(pjoin(tests_path, "..", "..", "cddl_gen"))
```

Comment:
Can we avoid the path hack below now?

Reply:
It can, yes. I tried removed the `sys.path` hack and called the `cddl_gen` CLI entry point in the command list, and the tests passed.

Since CI is already installing the package as a build/installation check step, it would be available in the CI environment to do it this way.

Reply:
Additionally, during development when the package may not be officially installed, we can still utilize this via `python3 setup.py develop`. This will "install" the package in a way that the `cddl_gen` CLI entry point is still available and `cddl_gen` is importable, and will represent changes to the package files without having to reinstall.

It's a nice way to dev on an installable package, while also removing this hack here and making it both callable and importable by tests.

I can add an `ImportError` catch as well so that developers are notified to do it this way.